extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -567,7 +567,7 @@ export class AmpA4A extends AMP.BaseElement {
       this.adPromise_ = null;
       this.adUrl_ = null;
       this.creativeBody_ = null;
-      this.experimentalNonAmpCreativeRenderMethod_ = null;
+      this.nonAmpCreativeRenderMethod_ = null;
```

Comment:
Should this be the default (client cache)?   We should have a test where we create A4A Element, call unlayoutcallback, and verify state has not changed.